src/components/PollResultModal/PollResultChart/helpers.test.ts

```diff
@@ -30,4 +30,27 @@ describe('generateScale', () => {
     const result: Array<{ value: number }> = [];
     expect(generateScale(result)).toEqual([0]);
   });
+
+  it.only('should repeat the colors if the answers are more then 7', () => {
```

Comment:
This is not a test for `generateScale`. You would want to create a new `describe` block for the `createAnswersColorScale` function.